services/movie.py

```diff
@@ -0,0 +1,48 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None, actors_ids: list[int] = None
```

Comment:
Using mutable default arguments like `genres_ids: list[int] = None` and `actors_ids: list[int] = None` can lead to unexpected behavior. It's better to use `None` as the default value and initialize the list inside the function if needed.